util/memory/tracker.go

```diff
@@ -136,12 +137,9 @@ func (t *Tracker) ReplaceChild(oldChild, newChild *Tracker) {
 func (t *Tracker) Consume(bytes int64) {
 	var rootExceed *Tracker
 	for tracker := t; tracker != nil; tracker = tracker.parent {
-		tracker.Lock()
-		tracker.bytesConsumed += bytes
-		if tracker.bytesLimit > 0 && tracker.bytesConsumed >= tracker.bytesLimit {
+		if atomic.AddInt64(&tracker.bytesConsumed, bytes) >= tracker.bytesLimit && tracker.bytesLimit > 0 {
```

Comment:
Is it safe to mix the use of `atomic` and `lock`? Other places that read or write `bytesConsumed` still uses lock.

Reply:
My fault, now all places using `bytesConsumed` has been replaced by atomic operations.